src/app/rundown/components/rundown/rundown.component.html

```diff
@@ -8,7 +8,7 @@
       <button *ngIf="rundown?.isActive" (click)="deactivateRundown()">Deactivate Rundown</button>
     </div>
     <div class="segments">
-      <sofie-segment *ngFor="let segment of rundown?.segments" [segment]="segment" [isRundownActive]="rundown?.isActive ?? false" (setNextEvent)="setNext($event)"></sofie-segment>
+      <sofie-segment *ngFor="let segment of rundown?.segments" [segment]="segment" [isOnAir]="segment.isOnAir" [isRundownActive]="rundown?.isActive ?? false" (setNextEvent)="setNext($event)"></sofie-segment>
```

Comment:
This seems redundant:
`[segment]="segment" [isOnAir]="segment.isOnAir"`
since you are passing the same value twice.

I assume this has something to do with you wanting to only listen on `isOnAir` changes?

Reply:
This was a "hack" to let the component know about changes when a segment is putOnAir or is takenOffAir.
Finding a better data patch strategy might remove the need for it. I'll investigate how we can go about this.